header.html

```diff
@@ -0,0 +1,7 @@
+{{define "header"}}
+<nav class="light-blue lighten-1" role="navigation">
+    <div class="nav-wrapper container"><a id="logo-container" href="#" class="brand-logo">Trashies</a>
+        <a href="index.html" data-activates="nav-mobile" class="button-collapse"><i class="material-icons">menu</i></a>
+    </div>
+</nav>
+{{end}}
```

Comment:
Do we do newlines here?

Reply:
We probably should, I keep expecting the autoformat to take care of it.